### PR TITLE
Feature/local build

### PR DESCRIPTION
**Pull request by akuzminsky**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 sudo: required
 env:
   matrix:
-    - PLATFORM=centos DISTRO=el OS_VERSION=7 DOCKER_IMAGE=centos:centos7
-    - PLATFORM=debian DISTRO=debian OS_VERSION=jessie DOCKER_IMAGE=debian:jessie
-    - PLATFORM=ubuntu DISTRO=ubuntu OS_VERSION=trusty DOCKER_IMAGE=ubuntu:trusty
+    - PLATFORM=centos DISTRO=el OS_VERSION=7 DOCKER_IMAGE=centos:centos7 LABEL=7
```
Comment (ovaistariq) [inline .travis.yml]: LABEL is a duplicate of OS_VERSION which is already exported as an environment variable here.

File: Makefile [hunk 2]
```diff
@@ -164,35 +164,15 @@ docker-test: ## Test twindb-backup in a docker container
...
-		-e LOG_LEVEL=${LOG_LEVEL} \
-		-e OMNIBUS_BRANCH=${OMNIBUS_BRANCH} \
-		-e OMNIBUS_SOFTWARE_BRANCH=${OMNIBUS_SOFTWARE_BRANCH} \
-		-v ${pwd}/pkg:/twindb-backup/omnibus/pkg \
-		-v ${pwd}/keys:/keys \
-		-v "${pwd}/cache/${PLATFORM}:/var/cache/omnibus" \
-		"twindb/backup-omnibus-${PLATFORM}"
+	@docker run \
+		-v ${pwd}:/twindb-backup \
+		"twindb/omnibus-${PLATFORM}:${LABEL}" \
```
Comment (ovaistariq) [inline Makefile]: ${PLATFORM}:${OS_VERSION}

File: omnibus/omnibus_build.sh [hunk 3]
```diff
@@ -1,52 +1,5 @@
...
-# the extra package files are destroyed when the build container stops (we have
-# to tweak omnibus-git-cache directly for that).
-git --git-dir=/var/cache/omnibus/cache/git_cache/opt/twindb-backup tag -d `git --git-dir=/var/cache/omnibus/cache/git_cache/opt/twindb-backup tag -l | grep twindb-backup` || true
-
-# Install the gems we need, with stubs in bin/
-cd omnibus
-bundle update
-bin/omnibus build -l=$LOG_LEVEL $PROJECT_NAME
+cd /twindb-backup/omnibus
+bundle install --binstubs
```
Comment (ovaistariq) [inline omnibus/omnibus_build.sh]: `bundle update` so that the gems already installed are updated.

File: omnibus/omnibus_build.sh [hunk 4]
```diff
@@ -1,52 +1,5 @@
...
+omnibus build twindb-backup
```
Comment (ovaistariq) [inline omnibus/omnibus_build.sh]: prefix command with `bin/` so that global omnibus is never used.

File: omnibus/omnibus_build.sh [hunk 5]
```diff
@@ -1,52 +1,6 @@
 #!/bin/bash -e
 
-###########################
```
Comment (ovaistariq) [inline omnibus/omnibus_build.sh]: As the comment suggests, we should rebuild the docker image as well.

Comment (akuzminsky) [inline omnibus/omnibus_build.sh]: Why?

Comment (ovaistariq) [inline omnibus/omnibus_build.sh]: To clean it up because you no longer rely on the git checkout in the image.

Comment (akuzminsky) [inline omnibus/omnibus_build.sh]: But it doesn't use old images See https://hub.docker.com/u/twindb/dashboard/

Comment (ovaistariq) [inline omnibus/omnibus_build.sh]: I didn't realize you had already updated the Dockerfiles :)

**Pull request merged**